python/kwiver/sprokit/processes/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+
+kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/kwiver_process.py
+  sprokit/processes
+  kwiver_process )
+
+kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/ApplyDescriptor.py
+  sprokit/processes
+  ApplyDescriptor )
+
+kwiver_add_python_module( ${CMAKE_CURRENT_SOURCE_DIR}/homography_writer.py
+  sprokit/processes
+  homography_writer )
```

Comment:
There is a difference in naming styles for python processes. Some are camel case, some are snake case. Is there a preferred naming style?

Reply:
snake case should always be preferred unless there is an existing codebase with a different convention.

Reply:
Fixed in 590db6c666bbbcc5c291526177776e9da0ff854b and c4d668b5e84635844eb5fa8e460a72616555dba8